components/proxy/conf/Caddyfile

```diff
@@ -139,6 +139,13 @@
 	}
 }
 
+# Internal analytics endpoint
+:9546 {
```

Comment:
For reviewers, this is private only because by default we only expose 80/443 to the public internet though the Loadbalancer config.

Does that assumption hold generally even in dedicated environments?

Reply:
I would think Dedicated has similar AWS lB config. cc @iQQBot maybe you know better